cmd/finch/support_bundle.go

```diff
@@ -0,0 +1,103 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/runfinch/finch/pkg/command"
+	"github.com/runfinch/finch/pkg/flog"
+	"github.com/runfinch/finch/pkg/lima"
+	"github.com/runfinch/finch/pkg/support"
+)
+
+func newSupportBundleCommand(logger flog.Logger, builder support.BundleBuilder, lcc command.LimaCmdCreator) *cobra.Command {
+	supportBundleCommand := &cobra.Command{
+		Use:   "support-bundle",
+		Short: "Support bundle management",
+	}
+	supportBundleCommand.AddCommand(
+		newSupportBundleGenerateCommand(logger, builder, lcc),
+	)
+	return supportBundleCommand
+}
+
+func newSupportBundleGenerateCommand(logger flog.Logger, builder support.BundleBuilder, lcc command.LimaCmdCreator) *cobra.Command {
+	supportBundleGenerateCommand := &cobra.Command{
+		Use:   "generate",
+		Args:  cobra.NoArgs,
+		Short: "Generate support bundle",
+		Long:  "Generates a collection of logs and configs that can be uploaded to a Github issue to help debug issues.",
+		RunE:  newGenerateSupportBundleAction(logger, builder, lcc).runAdapter,
+	}
+
+	supportBundleGenerateCommand.Flags().StringArray("include", []string{},
+		"additional files to include in the support bundle, specified by absolute or relative path")
+	supportBundleGenerateCommand.Flags().StringArray("exclude", []string{},
```

Comment:
Where is the best place to state "exclude" has higher privilege than "include"? Maybe specifiy in both --exclude and --include?